src/main/scala/org/apache/mesos/chronos/scheduler/config/MainModule.scala

```diff
@@ -48,17 +50,29 @@ class MainModule(val config: SchedulerConfiguration) extends AbstractModule {
   @Provides
   def provideFrameworkInfo(frameworkIdUtil: FrameworkIdUtil): FrameworkInfo = {
     import mesosphere.util.BackToTheFuture.Implicits.defaultTimeout
+    import scala.concurrent.ExecutionContext.Implicits.global
 
-import scala.concurrent.ExecutionContext.Implicits.global
-
-    val frameworkInfo = FrameworkInfo.newBuilder()
+    val frameworkInfoBuilder = FrameworkInfo.newBuilder()
       .setName(config.mesosFrameworkName())
       .setCheckpoint(config.mesosCheckpoint())
       .setRole(config.mesosRole())
       .setFailoverTimeout(config.failoverTimeoutSeconds())
       .setUser(config.user())
-    frameworkIdUtil.setIdIfExists(frameworkInfo)
-    frameworkInfo.build()
+
```

Comment:
This is a nit, but let's but the else if and else should be one line up. See another example in the Chronos code:

//If the job was deleted after the taskId was added to the queue, the task could be empty.
      if (jobOption.isEmpty) {
        //remove invalid task
        removeTask(taskId)
        None
      } else if (jobOption.get.disabled) {
        None
      } else {
        Some(taskId, job)
      }